### DISCUSSION TITLE
FAQ

### DISCUSSION THREAD

**Discussion by christoph2** [christoph2/pyA2L#33] [General]
Just started working on a FAQ document.

Your questions are highly welcome 🤗

**Comment by christoph2** [comment]
There it is: the initial [FAQ](https://github.com/christoph2/pyA2L/blob/master/docs/faq.rst) document, containing three QAs

**Comment by karsten-burger** [comment]
Hello Christoph, using Google I found a document: https://pya2l.readthedocs.io/_/downloads/en/latest/pdf/  of Mar 27, 2020.
Is this the current version? Maybe you could add a link in your README or in the project description, I did not quite understand how to start and where to find documentation.
Great idea, the ASAP2 Lib is REALLY expensive, only the lib of jnachbaur.de would be within reach for me.

**Comment by KarthikVeluswamy** [comment]
Can I use your library to edit A2L files ? I just need to replace signal names with the signal description, where the name was randomly generated buy the tool chain and the developer used the comment section to identify the signals.

**Comment by CrazyLMY** [comment]
Hello Christoph, I wonder if pya2l tool can generate IF_DATA in Characteristic or Measurement?... I have tried for a long time because my poor python ability....
Waiting to hear from you...

**Comment by neethuabraham2025** [comment]
@christoph2 I have a question. How can I export an a2l file using pya2l? for me db.export() is not working.
